### DISCUSSION TITLE
[General] How to check the Allocation for dApps in the airdrop ZK?

### DISCUSSION THREAD

**Discussion by nicolasmarin** [zkSync-Community-Hub/zksync-developers#579] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hi everyone,

I'm looking for information on how to check the allocation for dApps in the zkSync Era airdrop. Could someone guide me on where to find this information or the steps I need to follow? Any help would be greatly appreciated.

Thanks!

### Repo Link (Optional)

https://github.com/nfts2me

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

For your question in particular, you can find the full list of eligible dApp here: https://github.com/ZKsync-Association/zknation-data/blob/main/zksync_native_project_list.csv

**Reply by nicolasmarin** [reply]
Hello,

I can't find the eligibility criteria for dApps:
https://docs.zknation.io/zk-token/zk-token-faq

**Reply by 0xsisyfos** [reply]
We did not publish eligibility criterias for Dapp. Only the list of eligible Dapps.
